packages/app/src/elements/import-dialog.ts

```diff
@@ -224,35 +235,14 @@ export class ImportDialog extends Dialog<File, void> {
         const importFormat = (await imp.guessFormat(file)) || imp.CSV;
         this._formatSelect.value = importFormat.value;
 
-        // Select first options properly
-        if (importFormat.value === imp.CSV.value) {
-            if (
-                this._nameColumnSelect &&
-                (this._nameColumnSelect.selectedIndex === -1 || this._nameColumnSelect.selectedIndex === undefined)
```

Comment:
This conditional is inappropriate since we're resetting the value so we want to we don't care if another value had been selected previously.

Reply:
Gotcha.